### ISSUE TITLE
KeyError: 'log_directory'

### ISSUE THREAD

**Issue by apeeters** [danielwelch/hassio-zigbee2mqtt#52]
When the (optional) log_directory parameter is missing I get this error in the logs:
```
Traceback (most recent call last):
  File "set_config.py", line 97, in <module>
    main(args.options_path, args.data_path)
  File "set_config.py", line 81, in main
    config.set_log_dir(data_path, options['log_directory'])
KeyError: 'log_directory'
```

**Comment by zen** [comment]
Same here

**Comment by nismanoke** [comment]
Same issue as i have #53

**Comment by danielwelch** [comment]
this should be fixed in latest release v0.1.5.1
